### PR TITLE
[Documentation] Add link to 'custom adapter'

### PR DESCRIPTION
**Pull request by angelayanpan**
## Description

Add link to 'custom adapter'

### CODE REVIEW

File: packages/store/addon/-private/store-service.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ export interface CreateRecordProperties {
   }
   ```
 
-  You can learn more about writing a custom adapter by reading the `Adapter`
+  You can learn more about writing a custom adapter by reading the [Adapter](https://guides.emberjs.com/release/models/customizing-adapters/)
```
Comment (runspired) [inline packages/store/addon/-private/store-service.ts:131]: thanks! probably the adapter docs would be better, the guides are very outdated:

- implementation details: https://api.emberjs.com/ember-data/release/classes/MinimumAdapterInterface
- overview: https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fadapter

Comment (angelayanpan) [reply packages/store/addon/-private/store-service.ts]: hi thanks! @runspired  

I changed the link to the overview link that you shared.

**Review by runspired — APPROVED** [review]

**Review by runspired — CHANGES_REQUESTED** [review]
just realized this is targeted at "release", could you target to "main" instead?

**Comment by runspired** [comment]
it looks as though this change is no longer necessary

**Pull request closed without merging**